src/apiCalls.js

```diff
@@ -0,0 +1,14 @@
+export const fetchDataFor = (dataType) => { 
+    return fetch(`http://localhost:3001/api/v1/${dataType}`)
+    .then(response => {
+    if (!response.ok) {
+        throw new Error('Error with requesting information.');
+      } else {
+       return response.json();
+      }
+    }).catch(error => {
+        error = 'TypeError: Failed to fetch' ? errorMsg.innerText = 'Server is not currently running' : 
+        errorMsg.innerText = 'Error encountered. See console.'
+        console.warn(error)
```

Comment:
I'm confused by what's happening here. You're assigning error to a string and also errorMsg to a string based of a ternary?¿